src/features/applications/hooks/defaults.ts

```diff
@@ -14,6 +15,15 @@ hooks.addPureHook('POST', 'resin', 'application', {
 			);
 		}
 
+		request.values.uuid =
+			request.values.uuid || (await pseudoRandomBytesAsync(16)).toString('hex');
```

Comment:
That's not strictly a valid uuid btw, since uuid v4 has some header info around version stuff I think

Reply:
true... we could be strict and enforce a fully valid UUIDv4.